ldms/src/sampler/zfs_leafvdevs/zfs_leafvdevs.man

```diff
@@ -0,0 +1,62 @@
+.\" Manpage for Plugin_procnetdev2
```

Comment:
Rewrite for your plugin. Also, please add "Plugin_" to the front of the man page file name. For whatever reason, that is the ldms Way.